Getting some error building this?

```
make build
mkdir -p dist
echo "/* cBioPortal Clinical Timeline v0.0.20-2-ge763c28 | Maintained @ github.com/cbioportal/clinical-timeline */" > dist/clinical-timeline.js
sed -e '/start-test-code-not-included-in-build/','/end-test-code-not-included-in-build/d' js/lib/d3-timeline.js js/plugins/configButton.js js/plugins/exportTimeline.js js/plugins/helperLines.js js/plugins/overviewAxis.js js/plugins/pluginPrototype.js js/plugins/trimTimeline.js js/plugins/util.js js/plugins/verticalLine.js js/plugins/zoom.js js/clinicalTimeline.js js/parser.js js/sanity.js >> dist/clinical-timeline.js
echo "/* cBioPortal Clinical Timeline v0.0.20-2-ge763c28 | Maintained @ github.com/cbioportal/clinical-timeline */" > dist/clinical-timeline.min.js
if [ ! -d "node_modules" ]; then \
                echo "installing node_modules"; \
                npm install;  \
        fi
uglifyjs dist/clinical-timeline.js >> dist/clinical-timeline.min.js
Parse error at dist/clinical-timeline.js:1788,13
      return `${time.y}y${Math.abs(time.m)}m${Math.abs(time.d)}d`;
             ^
SyntaxError: Unexpected character '`'
    at JS_Parse_Error.get (eval at <anonymous> (/Users/debruiji/git/clinical-timeline/node_modules/uglify-js/tools/node.js:27:1), <anonymous>:86:23)
    at /Users/debruiji/git/clinical-timeline/node_modules/uglify-js/bin/uglifyjs:384:40
    at time_it (/Users/debruiji/git/clinical-timeline/node_modules/uglify-js/bin/uglifyjs:620:15)
    at /Users/debruiji/git/clinical-timeline/node_modules/uglify-js/bin/uglifyjs:345:9
    at FSReqWrap.readFileAfterClose [as oncomplete] (fs.js:511:3)
make: *** [dist/clinical-timeline.min.js] Error 1
```
Maybe I'm doing something wrong

--- a/index.html
+++ b/index.html
@@ -1,10 +1,10 @@
 <!doctype html>
 <html>
 <head>
-  <script src="//code.jquery.com/jquery-latest.min.js"></script>
+  <script src="http://code.jquery.com/jquery-latest.min.js"></script>
   <script src="js/lib/jquery.dataTables.min.js"></script>
   <script src="js/lib/jquery.qtip.min.js"></script>
-  <script src="//d3js.org/d3.v3.min.js" charset="utf-8"></script>
+  <script src="http://d3js.org/d3.v3.min.js" charset="utf-8"></script>
   <script src="js/lib/underscore-min.js"></script>
   <script>
   /**
@@ -25,11 +25,11 @@
    */
   Object.setPrototypeOf = Object.setPrototypeOf || function(obj, proto) {
     obj.__proto__ = proto;
-    return obj;
+    return obj; 
   }
   </script>
-  <script src="//cdn.rawgit.com/MrRio/jsPDF/v1.3.5/dist/jspdf.min.js"></script>
-  <script src="//html2canvas.hertzen.com/build/html2canvas.js"></script>
+  <script src="http://cdn.rawgit.com/MrRio/jsPDF/v1.3.5/dist/jspdf.min.js"></script>
+  <script src="http://html2canvas.hertzen.com/build/html2canvas.js"></script>
   <script src="js/lib/d3-timeline.js"></script>
   <script src="js/plugins/util.js"></script>
   <script src="js/plugins/pluginPrototype.js"></script>
@@ -43,8 +43,8 @@
   <script src="js/clinicalTimeline.js"></script>
   <script src="js/sanity.js"></script>
   <script src="js/parser.js"></script>
-  <link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
-  <script src="//maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
+  <link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css">
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
   <link href="css/lib/jquery.qtip.min.css" type="text/css" rel="stylesheet" />
   <link href="css/lib/dataTables.tableTools.min.css" type="text/css" rel="stylesheet" />
   <style type="text/css">
@@ -173,7 +173,7 @@
       var testData = getURLParameterByName("test");
       var dataFed = "data1";
       var waitForParse = false;
-
+      
       if (jsonURL !== "") {
         if (jsonURL.indexOf("?view=") > -1) {
           jsonURL = jsonURL.substring(0, jsonURL.indexOf("?view="));
@@ -225,42 +225,42 @@
         .enableTrackTooltips(true)
         .plugins([
           {
-            obj: new trimClinicalTimeline("Trim Timeline"),
+            obj: new trimClinicalTimeline("Trim Timeline"), 
             enabled: trimmingForSceenshotTest
-          },
+          },  
           {
-            obj: new clinicalTimelineHelperLines("Helper Lines"),
+            obj: new clinicalTimelineHelperLines("Helper Lines"), 
             enabled: false
-          },
+          }, 
           {
-            obj: new clinicalTimelineZoom("Zoom"),
+            obj: new clinicalTimelineZoom("Zoom"), 
             enabled: !trimmingForSceenshotTest
           },
           {
-            obj: new clinicalTimelineOverviewAxis("Overview Axis"),
+            obj: new clinicalTimelineOverviewAxis("Overview Axis"), 
             enabled:!trimmingForSceenshotTest
           },
           {
-            obj: new clinicalTimelineVerticalLine("Vertical Timeline",
+            obj: new clinicalTimelineVerticalLine("Vertical Timeline", 
               {
-                tooltipControllerId: "#tooltip-controller",
-                hoverBegin: 200,
+                tooltipControllerId: "#tooltip-controller", 
+                hoverBegin: 200, 
                 hoverEnd: 770
               }),
             enabled: false
           },
           {
-            obj: new configCheckManager(null,
+            obj: new configCheckManager(null, 
               {
                 configUlId: "#timeline-wrapper #config-div #config-dropdown"
-              }),
+              }), 
             enabled: true
           },
           {
-            obj: new clinicalTimelineExporter("Export",
+            obj: new clinicalTimelineExporter("Export", 
               {
                 exportDivId : "#export-div",
-              }),
+              }), 
             enabled: true
           }
         ])
@@ -287,7 +287,7 @@
       <span class="caret"></span></button>
       <ul class="dropdown-menu">
         <script type="text/javascript">
-          var exporter = (new clinicalTimelineExporter()).run(null, {exportDivId : "#export-div", timelineDiv : "#timeline"});
+          var exporter = (new clinicalTimelineExporter()).run(null, {exportDivId : "#export-div", timelineDiv : "#timeline"}); 
         </script>
         <li><a href='javascript:exporter.generateSVG()'>SVG Image</a></li>
         <li><a href='javascript:exporter.generatePNG(true)'>PNG Image</a></li>

--- a/js/clinicalTimeline.js
+++ b/js/clinicalTimeline.js
@@ -2,6 +2,53 @@
 /* start-test-code-not-included-in-build */
 d3 = require('d3');
 /* end-test-code-not-included-in-build */
+
+formatDayMonthYear = function(time) {
+  if (Math.abs(time.m) === 12) {
+    time.y = time.y + time.m / 12;
+    time.m = 0;
+  }
+
+  if (time.y !== 0) {
+    if (time.m !== 0) {
+      if (time.d !== 0) {
+        return time.y + "y" + Math.abs(time.m) + "m" + Math.abs(time.d) + "d";
+      } else {
+        return time.y + "y" + Math.abs(time.m) + "m";
+      }
+    } else if (time.d !== 0) {
+      return time.y + "y" + Math.abs(time.d) + "d";
+    } else {
+      return time.y + "y";
+    }
+  } else if (time.m !== 0) {
+    if (time.d !== 0) {
+      return time.m + "m" + Math.abs(time.d) + "d";
+    } else {
+      return time.m + "m";
+    }
+  } else {
+    return time.d + "d";
+  }
+}
+
+formatMonthYear = function(time) {
+  if (Math.abs(time.m) === 12) {
+    time.y = time.y + time.m / 12;
+    time.m = 0;
+  }
+  
+  if (time.y !== 0) {
+    if (time.m !== 0) {
+      return time.y + "y" + Math.abs(time.m) + "m";
+    } else {
+      return time.y + "y";
+    }
+  } else {
+    return time.m + "m";
+  }
+}
+
 var clinicalTimeline = (function(){
   "use strict";
 
@@ -30,6 +77,7 @@ var clinicalTimeline = (function(){
     chart = null,
     zoomStart = null,
     trimmed = false,
+    trimmingDidNothing = false,
     clinicalTimelinePlugins,
     clinicalTimelineReadOnlyVars,
     fractionTimelineShown = 1.0;
@@ -63,7 +111,7 @@ var clinicalTimeline = (function(){
     }
     
     minDays = Math.min(getMinStartingTime(allData), 0);
-    var zoomLevel = timeline.computeZoomLevel(minDays, maxDays, width * zoomFactor),
+    var zoomLevel = timeline.computeZoomLevel(minDays, maxDays, width * zoomFactor, timeline.fractionTimelineShown()),
       tickValues = timeline.getTickValues(minDays, maxDays, zoomLevel);
     
     beginning = tickValues[0];
@@ -753,12 +801,10 @@ var clinicalTimeline = (function(){
             case "days":
             case "3days":
             case "10days":
-              d = time.toDays();
-              dayFormat = d + "d";
+              dayFormat = formatDayMonthYear(time);
               break;
             case "months":
-              m = time.m + 12 * time.y;
-              dayFormat = m + "m";
+              dayFormat = formatMonthYear(time);
               break;
             case "years":
               y = time.y;
@@ -780,9 +826,9 @@ var clinicalTimeline = (function(){
    * @param  {Number} width
    * @return {string} zoomLevel
    */
-  timeline.computeZoomLevel = function(minDays, maxDays, width) {
+  timeline.computeZoomLevel = function(minDays, maxDays, width, fractionTimelineShown) {
     var pixelsPerDay = parseFloat(parseInt(width) / difference(parseInt(minDays), parseInt(maxDays)));
-    pixelsPerDay = pixelsPerDay / timeline.fractionTimelineShown();
+    pixelsPerDay = pixelsPerDay / fractionTimelineShown;
     if (pixelsPerDay < 1) {
       return "years";
     } else if (pixelsPerDay < 10){
@@ -797,19 +843,25 @@ var clinicalTimeline = (function(){
   }
 
   timeline.approximateTickToDayValue = function approximateTickToDayValue(tick) {
-    var dayRegex = /(-?\d+)(:?d)/
-    var monthRegex = /(-?\d+)(:?m)/
-    var yearRegex = /(-?\d+)(:?y?)/
-  
-    if (tick.match(dayRegex)) {
-      return parseInt(tick.match(dayRegex)[1])
+    var dayRegex = /(-?)((\d+)(?:y))?((\d+)(?:m))?((\d+)(?:d))/
+    var monthRegex = /(-?)((\d+)(?:y))?((\d+)(?:m))/
+    var yearRegex = /(-?)((\d+)(?:y))/
+
+    var uParse = function (num) { return num === undefined ? 0 : parseInt(num); };
+
+    var match = tick.match(dayRegex);
+    if (match) {
+      return uParse(match[1] + "1") * (uParse(match[3]) * 365 + uParse(match[5]) * 30.5 + uParse(match[7]))
     }
-    if (tick.match(monthRegex)) {
-      return parseInt(tick.match(monthRegex)[1]) * 30.5 
+    match = tick.match(monthRegex);
+    if (match) {
+      return uParse(match[1] + "1") * (uParse(match[3]) * 365 + uParse(match[5]) * 30.5)
     }
-    if (tick.match(yearRegex)) {
-      return parseInt(tick.match(yearRegex)[1]) * 365
+    match = tick.match(yearRegex);
+    if (match) {
+      return uParse(match[1] + "1") * (uParse(match[3]) * 365)
     }
+    return 0;
   }
 
   /**
@@ -821,7 +873,7 @@ var clinicalTimeline = (function(){
    * @param  {Number} width
    * @return {Number} zoomFactor    
    */
-  timeline.computeZoomFactor = function(zoomLevel, minDays, maxDays, width) {
+  timeline.computeZoomFactor = function(zoomLevel, minDays, maxDays, width) {    
     switch(zoomLevel) {
       case "years":
         return 0.9 * difference(parseInt(minDays), parseInt(maxDays)) / parseInt(width);
@@ -970,6 +1022,14 @@ var clinicalTimeline = (function(){
       return trimmed
     }
     trimmed = update;
+    return timeline;
+  }
+
+  timeline.trimmingDidNothing = function (update) {
+    if (!arguments.length) {
+      return trimmingDidNothing
+    }
+    trimmingDidNothing = update;
     return timeline;
   }
 

--- a/js/plugins/overviewAxis.js
+++ b/js/plugins/overviewAxis.js
@@ -21,14 +21,24 @@ clinicalTimelineOverviewAxis.prototype.run = function(timeline, spec) {
     zoomFactor = timeline.zoomFactor(),
     divId = timeline.divId(),
     overviewSVG = d3.select(divId+" .overview"),
-    originalZoomLevel = timeline.computeZoomLevel(readOnlyVars.minDays, readOnlyVars.maxDays, timeline.width()),
+    originalZoomLevel = timeline.computeZoomLevel(
+      readOnlyVars.minDays,
+      readOnlyVars.maxDays,
+      timeline.width(),
+      timeline.fractionTimelineShown()
+    ),
     overviewAxisTicks = timeline.getTickValues(readOnlyVars.minDays, readOnlyVars.maxDays, originalZoomLevel),
     minDayTick = overviewAxisTicks[0],
     maxDayTick =  overviewAxisTicks[overviewAxisTicks.length-1],
     overViewScale = d3.time.scale()
       .domain([clinicalTimelineUtil.roundDown(readOnlyVars.minDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel)), clinicalTimelineUtil.roundUp(readOnlyVars.maxDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel))])
       .range([0 + margin.overviewAxis.left, overviewAxisWidth - margin.overviewAxis.right]),
-    zoomLevel = timeline.computeZoomLevel(readOnlyVars.minDays, readOnlyVars.maxDays, timeline.width() * zoomFactor),
+    zoomLevel = timeline.computeZoomLevel(
+      readOnlyVars.minDays,
+      readOnlyVars.maxDays,
+      timeline.width() * zoomFactor,
+      timeline.fractionTimelineShown()
+    ),
     startAllowedOverview = overViewScale(clinicalTimelineUtil.roundDown(readOnlyVars.minDays, clinicalTimelineUtil.getDifferenceTicksDays(zoomLevel))) - overViewScale(clinicalTimelineUtil.roundDown(readOnlyVars.minDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel))),
     endAllowedOverview = overViewScale(clinicalTimelineUtil.roundUp(readOnlyVars.maxDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel))) - overViewScale(clinicalTimelineUtil.roundUp(readOnlyVars.maxDays, clinicalTimelineUtil.getDifferenceTicksDays(zoomLevel)));
 

--- a/js/plugins/trimTimeline.js
+++ b/js/plugins/trimTimeline.js
@@ -25,7 +25,7 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
     minDays = timeline.getReadOnlyVars().minDays,
     width = timeline.zoomFactor() > 1 ? timeline.zoomFactor() * timeline.width() : timeline.width(),
     margin = timeline.getReadOnlyVars().margin,
-    zoomLevel = timeline.computeZoomLevel(minDays, maxDays, width),
+    zoomLevel = timeline.computeZoomLevel(minDays, maxDays, width, timeline.fractionTimelineShown()),
     tickValues = timeline.getTickValues(minDays, maxDays, zoomLevel),
     kinkLineData = [{ "x": 75,  "y": 0  }, { "x": 80,  "y": 5 }, //drawing the kink svg
                     { "x": 85,  "y": -5 }, { "x": 90,  "y": 5 },
@@ -42,7 +42,17 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
     // the first and last ticks on the timeline, as ints
     ticksToShow = [tickValuesInt[0], tickValuesInt[tickValuesInt.length - 1]],
     expandedXAxis = d3.select(timeline.divId()+" > svg > g > g.axis");
+  
+  if (timeline.trimmingDidNothing()) {
+    // if the first trim, which happens when not zoomed
+    // does nothing, future trims should also do nothing.
+    // without this, there is a chance that they do, as zooming in
+    // changes the precision of the tick values, which decreases the duration
+    // needed to trigger a trim
+    return;
+  }
 
+  
   expandedXAxis.style("visibility", "hidden");
 
   var select = timeline.zoomFactor() > 1 ?
@@ -107,7 +117,6 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
 
   ticksToShow.sort(function(a, b) { return a - b; });
 
-  timeline.fractionTimelineShown(ticksToShow.length / tickValues.length);
   // redetermine breakTimelineForKinkIndex after sorting
   breakTimelineForKinkIndex = [];
   for (var i = 0; i < breakTimelineForKink.length; i++) {
@@ -254,6 +263,11 @@ trimClinicalTimeline.prototype.run = function (timeline, spec) {
     d3.select(divId).style("visibility", "hidden");
     timeline();
     d3.select(divId).style("visibility", "visible");
+  }
+
+  timeline.fractionTimelineShown(ticksToShow.length / tickValues.length);
+  if (ticksToShow.length === tickValues.length) {
+    timeline.trimmingDidNothing(true);
   }
 }
 

--- a/js/plugins/zoom.js
+++ b/js/plugins/zoom.js
@@ -1,3 +1,5 @@
+var zoomCount = 0;
+
 /**
  * Plugin to add rectangular zoom selection. 
  * Use brush to zoom. After zooming in, scroll mouse or drag to pan.
@@ -8,10 +10,11 @@ function clinicalTimelineZoom(name, spec){
   this.id = "zoom";
 }
 
-function findNearestTickToLeft(position) {
+function findNearestTick(position) {
   var regex = /(:?translate.)(\d+)(:?.*)/;
   var ticks = document.getElementsByClassName("tick");
   var tick = ticks;
+  var distance = Number.MAX_SAFE_INTEGER;
 
   for (var i = 0; i < ticks.length; i++) {
     var newTick = ticks[i];
@@ -19,10 +22,11 @@ function findNearestTickToLeft(position) {
       continue;
     }
     var tickPosition = newTick.getAttribute("transform").match(regex)[2];
-    var distance = position - parseInt(tickPosition);
-    if (distance < 0) {
+    var newDistance = Math.abs(position - parseInt(tickPosition));
+    if (distance < newDistance) {
       return tick;
     } else {
+      distance = newDistance;
       tick = newTick;
     }
   }
@@ -57,7 +61,16 @@ clinicalTimelineZoom.prototype.run = function(timeline, spec) {
      * zoom in after brush ends
      */
     var brushend = function() {
-      var originalZoomLevel = timeline.computeZoomLevel(minDays, maxDays, width);
+      timeline.trimmed(false); // this will get switched back to true by timTimeline
+      var extendLeft = parseInt(d3.select(timeline.divId()+" .extent").attr("x"));
+      var extendRight = extendLeft + parseInt(d3.select(timeline.divId()+" .extent").attr("width"));
+      var startTick = findNearestTick(extendLeft);
+      var endTick = findNearestTick(extendRight);
+      var zoomStart = timeline.approximateTickToDayValue(startTick.textContent)
+      var zoomEnd = timeline.approximateTickToDayValue(endTick.textContent)
+      timeline.zoomStart(zoomStart);
+
+      var originalZoomLevel = timeline.computeZoomLevel(minDays, maxDays, width, timeline.fractionTimelineShown());
       //handle positioning of the overview rectangle post zoom in.
       var overViewScale = d3.time.scale()
         .domain([roundDown(minDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel)), roundUp(maxDays, clinicalTimelineUtil.getDifferenceTicksDays(originalZoomLevel))])
@@ -68,17 +81,13 @@ clinicalTimelineZoom.prototype.run = function(timeline, spec) {
       var xDaysRect = brush.extent()[0].valueOf();
       timeline.zoomFactor((parseInt(width) - parseInt(margin.left) - parseInt(margin.right)) / (parseInt(d3.select(timeline.divId()+" .extent").attr("width"))));
       if (timeline.zoomFactor() > 0) {
-        timeline.zoomFactor(Math.min(timeline.zoomFactor(), timeline.computeZoomFactor("days", minDays, maxDays, width)) / timeline.fractionTimelineShown());
+        timeline.zoomFactor(Math.min(timeline.zoomFactor(), timeline.computeZoomFactor("days", minDays, maxDays, width)));
       } else {
-       timeline.zoomFactor(timeline.computeZoomFactor("days", minDays, maxDays, width));
+        timeline.zoomFactor(timeline.computeZoomFactor("days", minDays, maxDays, width));
       }
+      timeline.zoomFactor(timeline.zoomFactor() * timeline.fractionTimelineShown());
 
-      var extendPosition = parseInt(d3.select(timeline.divId()+" .extent").attr("x"));
-      var tick = findNearestTickToLeft(extendPosition);
-      timeline.zoomStart(timeline.approximateTickToDayValue(tick.textContent));
-      timeline.trimmed(false);
-
-      var zoomLevel = timeline.computeZoomLevel(readOnlyVars.minDays, readOnlyVars.maxDays, timeline.width() * timeline.zoomFactor());
+      var zoomLevel = timeline.computeZoomLevel(zoomStart, zoomEnd, timeline.width(), 1);
       var tickValues = timeline.getTickValues(readOnlyVars.minDays, readOnlyVars.maxDays, zoomLevel);
       // TODO: Translation post zooming is not entirely correct
       if (xDaysRect > minDays) {
@@ -112,7 +121,7 @@ clinicalTimelineZoom.prototype.run = function(timeline, spec) {
       });
     };
 
-    if (timeline.computeZoomLevel(minDays, maxDays, width * timeline.zoomFactor()) !== "days") {
+    if (timeline.computeZoomLevel(minDays, maxDays, width * timeline.zoomFactor(), timeline.fractionTimelineShown()) !== "days") {
       // add brush overlay
       var xScale = d3.time.scale()
          .domain([beginning, ending])
@@ -138,7 +147,12 @@ clinicalTimelineZoom.prototype.run = function(timeline, spec) {
           d3.select(divId+" #timelineZoomExplanation").style("visibility", "hidden");
       });
     }
+    zoomCount = 0;
   } else {
+    zoomCount++;
+    if (zoomCount > 1) {
+      return;
+    }
     // Add panning explanation and visual indicator
     zoomExplanation(divId, svg, "Scroll/drag to move", "visible", 180);
     d3.select(divId + " svg").style("cursor", "move");
@@ -152,7 +166,7 @@ clinicalTimelineZoom.prototype.run = function(timeline, spec) {
    * @param  {number} pos        position of the explanation's text
    */
   function zoomExplanation(divId, svg, text, visibility, pos) {
-    d3.select(divId + " svg")
+    d3.select(divId + " svg g .brush")
       .insert("text")
       .attr("transform", "translate("+(parseInt(svg.attr("width"))-pos)+", "+parseInt(svg.attr("height")-5)+")")
       .attr("class", "timeline-label")

--- a/test/unit-tests/basic-test.js
+++ b/test/unit-tests/basic-test.js
@@ -61,12 +61,12 @@ describe('clinicalTimeline.formatTime', function() {
      expect("10d").to.equal(formatTime(daysToTimeObject(10), "days"));
      expect("10d").to.equal(formatTime(daysToTimeObject(10), "3days"));
      expect("10d").to.equal(formatTime(daysToTimeObject(10), "10days"));
-     expect("610d").to.equal(formatTime(daysToTimeObject(610), "days"));
-     expect("610d").to.equal(formatTime(daysToTimeObject(610), "3days"));
-     expect("610d").to.equal(formatTime(daysToTimeObject(610), "10days"));
+     expect("1y8m5d").to.equal(formatTime(daysToTimeObject(610), "days"));
+     expect("1y8m5d").to.equal(formatTime(daysToTimeObject(610), "3days"));
+     expect("1y8m5d").to.equal(formatTime(daysToTimeObject(610), "10days"));
 
      expect("0m").to.equal(formatTime(daysToTimeObject(10), "months"));
-     expect("20m").to.equal(formatTime(daysToTimeObject(610), "months"));
+     expect("1y8m").to.equal(formatTime(daysToTimeObject(610), "months"));
 
      expect("0y").to.equal(formatTime(daysToTimeObject(10), "years"));
      expect("1y").to.equal(formatTime(daysToTimeObject(610), "years"));

